components/bl-transitions-component/README.md

```diff
@@ -52,3 +52,15 @@ For example:
 <img alt="Close button logic" src="./example-images/close_button_logic.jpg" width="900" />
 
 In the preview, you now have the option to change the visibility of the component with animation by clicking on either the Close or Open button.
+
+### Adding component with dynamic content in transition
+It can come in handy when animating a block that requires some time to render completely. To good utilize the Collapse Top and Collapse Left effects, it's important to employ this guid.
```

Comment:
If you want to animate content that is rendered with a delay (for example, asynchronously loaded) follow the guide: